tests/components/AppliedFilters.test.tsx

```diff
@@ -84,7 +93,8 @@ describe('AppliedFilters', () => {
 
   it('Static filters are rendered', () => {
     render(<AppliedFilters />);
-    const staticFilterDisplayName = mockedState.filters.static[0].value as string;
+    const staticFilter = mockedState.filters.static[0].filter as FieldValueStaticFilter;
```

Comment:
we can't do `const staticFilter: FieldValueStaticFilter` instead of the assert?

Reply:
we can't set the type like that because the static filters state could store other types of static filters too. but, I made some changes that removed the new asserts